src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -182,11 +182,27 @@ displayUserError e = case e of
   BadRepositoryField err ->
     displayRepositoryError err
   NoLicenseSpecified ->
-    para (concat
-      ["No license specified in bower.json. Please add one. ",
-       "Distributing code without a license means that nobody ",
-       "will be able to (legally) use it."
-      ])
+    vcat $
+      [ para (concat
+          [ "No license is specified in bower.json. Please add one, using the "
```

Comment:
I would just add that we _recommend_ adding a LICENSE file, even if it isn't necessary.